app/code/Magento/Indexer/Console/Command/IndexerStatusCommand.php

```diff
@@ -59,15 +62,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $rows[] = $rowData;
         }
 
-        usort($rows, function ($comp1, $comp2) {
-            return strcmp($comp1['Title'], $comp2['Title']);
-        });
+        usort(
+            $rows,
+            function ($comp1, $comp2) {
```

Comment:
You can add `array` type hint to both of them

Reply:
added type hints within the last commit

Reply:
I'm sorry, I wasn't clear what I mean:
`function (array $comp1, array $comp2) {`